ocs_ci/deployment/baremetal.py

```diff
@@ -290,13 +290,12 @@ def deploy(self, log_cli_level="DEBUG"):
             )
             logger.info("Uploading PXE files")
             ocp_version = get_ocp_version()
-            float_ocp_version = float(ocp_version)
             for machine in self.mgmt_details:
                 if self.mgmt_details[machine].get("cluster_name") or self.mgmt_details[
                     machine
                 ].get("extra_node"):
                     pxe_file_path = self.create_pxe_files(
-                        ocp_version=float_ocp_version,
+                        ocp_version=ocp_version,
```

Comment:
should it be `Version.coerce(ocp_version)` ?

Reply:
no not needed here doing it here https://github.com/red-hat-storage/ocs-ci/pull/5279/files#diff-05053122d6e5936ca528fcd1643379f792a8350f094e1c0aa5c62903f86a8746R579